libs/wingc/src/type_check.rs

```diff
@@ -994,6 +996,9 @@ pub struct TypeChecker<'a> {
 	/// The path to the source file being type checked.
 	source_path: &'a Path,
 
+	/// JSII Manifest descriptions to be imported
+	jsii_imports: &'a mut Vec<JsiiImportSpec>,
```

Comment:
(random thoughts) It would be nice if there was some way to model that we want to allow this vec to be extended, but the individual `JsiiImportSpec` values are basically immutable / should not be touched. Not sure what the Rusty way to do this is though

Reply:
AFAIK the only way to do it would be to make the JsiiImportSpec fields private and add getters. or whatever this is doing https://github.com/dtolnay

Not sure i'm conviced it's worthwhile, but it's not that weird to do